src/server.c

```diff
@@ -78,6 +79,81 @@ drop_permissions(void)
 	}
 }
 
+static void seat_clear_touch_points(struct seat *seat,
```

Comment:
We don't support touch (yet), so don't need this function.

Reply:
:+1: